packages/renderer/src/lib/pvc/PVCList.svelte

```diff
@@ -0,0 +1,168 @@
+  if (selectedPVCs.length === 0) {
+    return;
+  }
+
+  // mark pvcs for deletion
+  bulkDeleteInProgress = true;
+  selectedPVCs.forEach(pvc => (pvc.status = 'DELETING'));
+  pvcs = pvcs;
+
+  await Promise.all(
+    selectedPVCs.map(async pvc => {
+      try {
+        await window.kubernetesDeletePersistentVolumeClaim(pvc.name);
+      } catch (e) {
+        console.error('error while deleting pvc', e);
+      }
+    }),
+  );
+  bulkDeleteInProgress = false;
+}
+
+let selectedItemsNumber: number;
+let table: Table;
+
+let statusColumn = new TableColumn<PVCUI>('Status', {
+  align: 'center',
+  width: '70px',
+  renderer: PVCColumnStatus,
+  comparator: (a, b) => a.status.localeCompare(b.status),
+});
+
+let nameColumn = new TableColumn<PVCUI>('Name', {
+  renderer: PVCColumnName,
+  comparator: (a, b) => a.name.localeCompare(b.name),
+});
+
+let storageClassColumn = new TableColumn<PVCUI, string>('Storage', {
+  renderMapping: pvc => pvc.storageClass,
+  renderer: TableSimpleColumn,
+  comparator: (a, b) => a.storageClass.localeCompare(b.storageClass),
+});
+
+let namespaceColumn = new TableColumn<PVCUI, string>('Namespace', {
+  renderMapping: pvc => pvc.namespace,
+  renderer: TableSimpleColumn,
+  comparator: (a, b) => a.namespace.localeCompare(b.namespace),
+});
+
+let accessModesColumn = new TableColumn<PVCUI>('Mode', {
+  renderer: PvcColumnMode,
+  overflow: true,
+  comparator: (a, b) => a.accessModes.join().localeCompare(b.accessModes.join()),
+});
+
+let sizeColumn = new TableColumn<PVCUI, string>('Size', {
+  renderMapping: pvc => pvc.size,
+  renderer: TableSimpleColumn,
+  comparator: (a, b) => a.size.localeCompare(b.size),
+});
+
+let ageColumn = new TableColumn<PVCUI, Date | undefined>('Age', {
+  renderMapping: pvc => pvc.created,
+  renderer: TableDurationColumn,
+  comparator: (a, b) => moment(b.created).diff(moment(a.created)),
+});
+
+const columns = [
+  statusColumn,
+  nameColumn,
+  namespaceColumn,
+  accessModesColumn,
+  storageClassColumn,
+  sizeColumn,
+  ageColumn,
+  new TableColumn<PVCUI>('Actions', { align: 'right', renderer: PVCColumnActions }),
+];
+
+const row = new TableRow<PVCUI>({ selectable: _pvc => true });
+</script>
+
+<NavPage bind:searchTerm="{searchTerm}" title="Persistent Volume Claims">
+  <svelte:fragment slot="additional-actions">
+    <KubeApplyYamlButton />
+  </svelte:fragment>
+
+  <svelte:fragment slot="bottom-additional-actions">
+    {#if selectedItemsNumber > 0}
+      <Button
+        on:click="{() => deleteSelectedPVCs()}"
+        title="Delete {selectedItemsNumber} selected items"
+        inProgress="{bulkDeleteInProgress}"
+        icon="{faTrash}" />
+      <span>On {selectedItemsNumber} selected items.</span>
+    {/if}
+    <div class="flex grow justify-end">
+      <KubernetesCurrentContextConnectionBadge />
+    </div>
+  </svelte:fragment>
+
+  <div class="flex min-w-full h-full" slot="content">
+    <Table
+      kind="pvc"
```

Comment:
Should be capitalized? i.e. should be 'No PVCs', 'Delete PVC', vs current 'No pvcs'.